packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -65,7 +88,7 @@ const TourGuide = () => {
             disableScrolling
             hideCloseButton
             disableCloseOnEsc
-            steps={tour_step_config}
+            steps={content_flag === ContentFlag.HIGH_RISK_CR ? high_risk_tour_step_config : tour_step_config}
```

Comment:
how we are going to handle steps for EU then?

Reply:
Added the new condition